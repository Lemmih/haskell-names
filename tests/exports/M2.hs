module M2 where

data A = A A
