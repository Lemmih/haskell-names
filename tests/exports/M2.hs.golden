Symbols (
  fromList
    [ SymConstructor
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M2"
                                          "A"
                          }
        , sv_fixity = Nothing
        , sv_typeName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M2"
                                          "A"
                          }
        }
    ] ) (
  fromList
    [ SymData
        { st_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M2"
                                          "A"
                          }
        , st_fixity = Nothing
        }
    ] )